Lesson32/event-and-state-app/Gokhan/src/components/Exercise10.js

```diff
@@ -0,0 +1,19 @@
+import { useEffect } from 'react';
+
+export const Exercise10 = () => {
+
+    const handleDoubleClick = () => {
+        console.log('The Button Was Double Clicked');
+    };
+
+    useEffect(() => {
+
+    }, []);
+
+    return (
+        <>
```

Comment:
You don't need the fragment component here since you only have one element (button).